Analytics-CSharp/Segment/Analytics/Compat/Migration.cs

```diff
@@ -0,0 +1,81 @@
+using System.Collections.Generic;
+using Segment.Analytics;
+using Segment.Serialization;
+
+namespace AnalyticsNetMigrationHelper {
+    public class Traits : Dictionary<string, object>
+    {
+    }
+
+    public class Properties  : Dictionary<string, object>
+    {
+    }
+
+    public static class AnalyticsExtensions
+    {
+        public static void Track(this Analytics analytics, string userId, string eventName)
+        {
+            analytics.Track(eventName, new JsonObject() { {"userId", userId} });
+        }
+        public static void Track(this Analytics analytics, string userId, string eventName, IDictionary<string, object> traits)
```

Comment:
use `Dictionary` instead of `IDictionary`, then we can pass it directly to the new api call.
```suggestion
        public static void Track(this Analytics analytics, string userId, string eventName, Dictionary<string, object> properties)
```

Reply:
I still get errors about both IDictionary and Dictionary not being convertable to ISerializable, and it doesn't want to even consider JsonObject.  Best alternative I see is:
`analytics.Track(eventName, new JsonObject(properties.ToDictionary(x=>x.Key, x=>x.Value as JsonElement)));`